### ISSUE TITLE
Add manual control toggle to main card interface

### ISSUE THREAD

**Issue by vertigo220** [Rem0o/FanControl.Releases#2594]
I sometimes want to just control the fans manually for various reasons, and it's a bit annoying having to open the expanded menu, enable manual control, adjust it, then expand the menu and disable it when I'm done, especially when I'm doing it with multiple fans. I would much prefer to simply have a toggle for it on the card. It could be done by clicking the image at the top-left, or it could be a little tabbed switch underneath next to the "Calibrated" tab/label, or the cards could simply be a little larger to accommodate it, as there's more than enough room. And, of course, there could be an option to show/hide it like many other things have.

**Comment by Rem0o** [comment]
It is a design choice to leave manual control as a secondary citizen. Intended usage of the software is of course fan curves.  

However, to accommodate your need, you could flip them all manual, and "save as" to a new configuration file, say "manual control.json". Whenever you want to flip it all manual, just switch config from the tray icon or CTRL+L in the UI.

**Comment by vertigo220** [comment]
I understand that, but as long as it's optional, it seems harmless enough to allow it to take a more prominent position. Just my thoughts. As for using the config file, problem with that is I wouldn't always want to do all or none. Perhaps another option would be adding the ability to select multiple fans at once, so I could just for example ctrl+click all the ones I want to change and change them all at once. This would also allow changing curves and other settings on multiple fans much quicker.

**Comment by Rem0o** [comment]
Isn't there any kind of pattern to your adjustments? Like isn't there a "group" of fans your are changing at the same time? You can just leave them all in manual mode all the time also, no need to toggle back.  You haven't really explained your use-case besides 
> "for various reasons"

 so it's hard for me to justify your request without understanding the reasons behind it.  Generally I want to understand the need before implementing a solution.

**Comment by vertigo220** [comment]
Not really. I sometimes need to do this if I want to turn one or more fans on/off/up/down for various reasons. I might want to temporarily decrease the noise, or I might be trying to find which specific fan(s) is/are making a certain noise, or play with the effects on noise and/or temperature with various adjustments, etc. Basically, the software is great for if you know exactly what fan speeds you want, but in order to determine that, it requires trial and error, which FC isn't really set up for very well, and even once it's set up, there are times one might want to make adjustments, whether temporary or for further testing, and it just requires more steps than it should to do this. And leaving it in manual mode all the time isn't good because that's not an ideal setup, and it's not making the most of the software. Ideally, you'd want it running on a curve most of the time, with the ability to make quick adjustments. Think of it like using a programmable thermostat. You wouldn't want to just use it on manual mode, holding the same temp all the time, but you also want it to be quick and easy, with minimal steps, to make adjustments as needed.
